required/latex-lab/testfiles-math-luatex/test-math-split-reqno-tbtags.tlg

```diff
@@ -114,42 +111,26 @@ Don't change this file in any respect.
 <p>98801701624C155B53FA609F5457AC26</p>^^M
 ^^M
 <math display="block" xmlns="http://www.w3.org/1998/Math/MathML">^^M
- <mtable displaystyle="true">^^M
+ <mtable class="gather" displaystyle="true" intent=":system-of-equations">^^M
  <mtr>^^M
- <mtd intent=":equationlabel">^^M
+ <mtd intent=":equation-label">^^M
  <mtext>^^M
 (3)^^M
  </mtext>^^M
  </mtd>^^M
- <mtd>^^M
- <mtable columnalign="right left" displaystyle="true">^^M
- <mtr>^^M
- <mtd>^^M
- <mi>𝑎</mi>^^M
- </mtd>^^M
- <mtd>^^M
- <mo lspace="0.278em" rspace="0.278em">=</mo>^^M
- <mi>𝑏</mi>^^M
+ <mtd intent=":pause-medium">^^M
+ <mi>^^M
+ <mglyph/>^^M
+ </mi>^^M
  </mtd>^^M
  </mtr>^^M
  <mtr>^^M
- <mtd>^^M
- </mtd>^^M
- <mtd>^^M
- <mo lspace="0.222em" rspace="0.222em">+</mo>^^M
- <mi>𝑐</mi>^^M
- </mtd>^^M
- </mtr>^^M
- </mtable>^^M
- </mtd>^^M
- </mtr>^^M
- <mtr>^^M
- <mtd intent=":equationlabel">^^M
```

Comment:
This kind of pattern (a lot of deletions) shows up a number of times ... deliberate?

Reply:
that are the broken test files from yesterday.

Reply:
so there needs to be some fix somewhere at some point?

Reply:
sorry ? No I simply corrected the faulty lvt's and now show the correct html and remade the tlg, so it should be fine.